pyproject.toml

```diff
@@ -0,0 +1,74 @@
+[tool.poetry]
+name = "target-hdfs"
+version = "1.0.0"
+description = "`target-hdfs` is a Singer target for hdfs, built with the Meltano Singer SDK."
+readme = "README.md"
+authors = ["Joao Amaral <joao.amaral@automattic.com>"]
+keywords = [
+    "ELT",
+    "hdfs",
+]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<4"
+singer-sdk = { version="~=0.34.1" }
+fs-s3fs = { version = "~=1.1.1", optional = true }
+requests = "~=2.31.0"
+# TODO: Set the tav v1.0.0 when it's released
+target-parquet = { git = "https://github.com/Automattic/target-parquet.git", branch = "target_parquet_1_0_0" }
```

Comment:
Why are we not releasing it?

Reply:
I'll release both together because there are some tests here that sometimes I need to include a log or fix a bug (like the one in the flatten_record).